docs/book/component-guide/model-deployers/vllm.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Deploying your LLM locally with vLLM.
+---
+
+# vLLM
+
+[vLLM](https://docs.vllm.ai/en/latest/) is a fast and easy-to-use library for LLM inference and serving.
+
+## When to use it?
+
+You should use vLLM Model Deployer:
+
+* Deploying Large Language models with state-of-the-art serving throughput creating an OpenAI-compatible API server
+* Continuous batching of incoming requests
+* Quantization: GPTQ, AWQ, INT4, INT8, and FP8
+* Features such as PagedAttention, Speculative decoding, Chunked pre-fill
+
+## How do you deploy it?
+
+The vLLM Model Deployer flavor is provided by the vLLM ZenML integration, so you need to install it on your local machine to be able to deploy your models. You can do this by running the following command:
+
+```bash
+zenml integration install vllm -y
+```
+
+To register the vLLM model deployer with ZenML you need to run the following command:
+
+```bash
+zenml model-deployer register vllm_deployer --flavor=vllm
+```
+
+The ZenML integration will provision a local vLLM deployment server as a daemon process that will continue to run in the background to serve the latest vLLM model.
+
+## How do you use it?
+
+If you'd like to see this in action, check out this example of a [deployment pipeline](https://github.com/zenml-io/zenml-projects/blob/79f67ea52c3908b9b33c9a41eef18cb7d72362e8/llm-vllm-deployer/pipelines/deploy_pipeline.py#L25).
```

Comment:
Note: Updating this url once https://github.com/zenml-io/zenml-projects/pull/131 is merged.